src/components/ReactTags.js

```diff
@@ -316,7 +316,7 @@ class ReactTags extends Component {
 
       if (
         (this.props.autocomplete === 1 && possibleMatches.length === 1) ||
-        (this.props.autocomplete === true && possibleMatches.length)
+        (this.props.autocomplete === true && possibleMatches.length === 1)
```

Comment:
can you remove this line and instead change line 318 to 
```
this.props.autocomplete && possibleMatches.length === 1
```
This will take care of both the cases.
Please add a spec for this change.